wordstats/language_data/t2l/code/README.md

```diff
@@ -0,0 +1,62 @@
+### cleaning_words.py
+
+If it is required to add words of a new language it is needed to follow the next steps:
+
+1. Download the aspell dictionary of that language (http://ftp.gnu.org/gnu/aspell/dict/) and follow the instructions. Basically you have to download the .tar.bz2 file (the 5 is the newest) and then:
+
+	-Unzip the folder
+
+	-Go with the terminal to that folder.
+
+	-Execute ./configure
+
+	-Execute make
+
+	-Execute make install
+
+	-Execute preunzip 'lang'.cwl (e.g: es.cwl)
+
+You will obtain a .wl file with the dictionnary.
+
+However there will be a script which will do this automatically, but if the result is not what you expect, then you can do it manually.
+
+2. It is needed to create a new .txt file and copy the dictionary. Python does not recognize .uw files. After that move the dictionary to the Dictionary folder.
+
```

Comment:
Hmmm I am not sure, because what the README does is explain every algorithm used. Maybe it is a bit confusing if I place it to the same level than `code` and `data` and someone enters in the README and sees the name of the algorithms and he/she won't understand a lot.

However what I can do is write an introduction saying that I am going to describe the algorithms in `code`. Do you like this idea?